app/views/job_invocations/welcome.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="blank-slate-pf">
+    <div class="blank-slate-pf-icon">
+        <%= icon_text("globe", "", :kind => "fa") %>
+    </div>
+    <h1><%= _('Job Invocations') %></h1>
+    <p>
+        <%= _("Foreman can run arbitrary commands on remote hosts using different providers, such as SSH or Ansible. Communication goes through the Smart Proxy so Foreman does not have to have direct access to the target hosts and can scale to control many hosts.") %></br>
+    </p>
+    <p><%= link_to _('Learn more about this in the documentation.'), 'https://www.theforeman.org/plugins/foreman_remote_execution/1.3/index.html' %></p>
```

Comment:
Please use the `documentation_url` helper, similar to https://github.com/theforeman/foreman_openscap/blob/0cf1573b1addd9f922b4bbcc298c476fe3535602/app/views/arf_reports/welcome.html.erb#L10